fe/src/com/baidu/palo/http/BaseAction.java

```diff
@@ -87,6 +86,7 @@ public void setQeService(QeService qeService) {
     @Override
     public void handleRequest(BaseRequest request) throws Exception {
         BaseResponse response = new BaseResponse();
+        LOG.debug("receive http request. url=", request.getRequest().uri());
```

Comment:
This log should remove

Reply:
this log is to show reqeust url to help debug.